man/ostree.repo-config.xml

```diff
@@ -5,6 +5,8 @@
 <!--
 Copyright 2014 Colin Walters <walters@verbum.org>
 
+SPDX-License-Identifier: LGPL-2.0+
```

Comment:
Hmm, actually...the man pages should probably be under the docs license.  But...let's not try to do that relicensing now.